### PR TITLE
Update supported databases in v19

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR updates the supported databases in v19. We also now list the databases that we support for importing.

### CODE REVIEW

File: content/en/docs/19.0/overview/supported-databases.md [hunk 1]
```diff
@@ -4,15 +4,23 @@ weight: 2
...
+
+## Supported Databases For Imports
 
-Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
+Vitess supports importing from a wide range of databases that include - 
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+- [MySQL](https://www.mysql.com/) version 5.7 to 8.0
+- [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) version 5.7 to 8.0
+- [MariaDB](https://mariadb.com)
```
Comment (GuptaManan100) [inline content/en/docs/19.0/overview/supported-databases.md]: I am unsure of the versions of MariaDB that we allow support from

Comment (GuptaManan100) [inline content/en/docs/19.0/overview/supported-databases.md]: Do we want to list anything else like RDS, etc?

Comment (mattlord) [reply content/en/docs/19.0/overview/supported-databases.md]: I don't think so. RDS has MySQL 8.0 e.g. so there's some implicit support there.

File: content/en/docs/19.0/overview/supported-databases.md [hunk 2]
```diff
@@ -4,15 +4,23 @@ weight: 2
...
 ---
 
+## Supported Databases In Vitess
+
 Vitess deploys, scales and manages clusters of open-source SQL database instances. Currently, Vitess supports the [MySQL](https://www.mysql.com/) and [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) databases.
 
 The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQL 8.0.
 
-## MySQL versions 5.7 to 8.0
+### MySQL versions 8.0
```
Comment (mattlord) [inline content/en/docs/19.0/overview/supported-databases.md]: We can make versions singular.

**Review by mattlord — COMMENTED** [review]
IMO we should also add warnings to the v17 and v18 docs that MySQL 5.7 support is EOL in v19.

File: content/en/docs/19.0/overview/supported-databases.md [hunk 3]
```diff
@@ -4,15 +4,23 @@ weight: 2
...
+
+Vitess supports the core features of MySQL versions 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 8.0.
```
Comment (deepthi) [inline content/en/docs/19.0/overview/supported-databases.md]: We can probably drop "versions" altogether, both in the heading and in the description.

File: content/en/docs/19.0/overview/supported-databases.md (hunk 1 above)
Comment (deepthi) [reply content/en/docs/19.0/overview/supported-databases.md]: Good question. @mattlord what should we say here?

**Comment by GuptaManan100** [comment]
I removed the `Versions` as you two pointed out. I have also added a line to both v18.0 and v17.0 docs to say that MySQL 5.7 is EOL and users should upgrade to 8.0 before upgrading to v19.

File: content/en/docs/19.0/overview/supported-databases.md (hunk 1 above)
Comment (mattlord) [reply content/en/docs/19.0/overview/supported-databases.md]: I'd say MariaDB 10.10+. If someone reports a failure with a later version then I'd consider it a bug.

Comment (GuptaManan100) [reply content/en/docs/19.0/overview/supported-databases.md]: Awesome possum. I'll make the change!

File: content/en/docs/17.0/overview/supported-databases.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users are requested to upgrade to 8.0 before upgrading to Vitess v19. {{< /info >}}
```
Comment (deepthi) [inline content/en/docs/17.0/overview/supported-databases.md]: I would make this stronger - active voice vs passive voice.
"users should upgrade to MySQL 8 before upgrading to Vitess v19."

8 versus 8.0 for consistency with previous sentence.

File: content/en/docs/18.0/overview/supported-databases.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users are requested to upgrade to 8.0 before upgrading to Vitess v19. {{< /info >}}
```
Comment (deepthi) [inline content/en/docs/18.0/overview/supported-databases.md]: I would make this stronger - active voice vs passive voice.
"users should upgrade to MySQL 8 before upgrading to Vitess v19."

8 versus 8.0 for consistency with previous sentence.

File: content/en/docs/17.0/overview/supported-databases.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ The [VTGate](../../concepts/vtgate/) proxy server advertises its version as MySQ
 
 Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
 
-{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended.{{< /info >}}
+{{< info >}}For new Vitess installations, MySQL or Percona Server for MySQL version 8 are recommended. MySQL version 5.7 is EOL in v19 and users should upgrade to MySQL 8 before upgrading to Vitess v19. {{< /info >}}
```
Comment (mattlord) [inline content/en/docs/17.0/overview/supported-databases.md]: Might be worth a separate warning block, but up to you. This is fine too IMO.

File: content/en/docs/19.0/overview/supported-databases.md [hunk 7]
```diff
@@ -4,15 +4,23 @@ weight: 2
...
 
-## MySQL versions 5.7 to 8.0
+### MySQL 8.0
+
+Vitess supports the core features of MySQL 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 8.0.
+
+## Supported Databases For Imports
 
-Vitess supports the core features of MySQL versions 5.7 to 8.0, with [some limitations](../../reference/compatibility/mysql-compatibility/). Vitess also supports [Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) versions 5.7 to 8.0.
+Vitess supports importing from a wide range of databases that include -
```
Comment (mattlord) [inline content/en/docs/19.0/overview/supported-databases.md]:
```suggestion
Vitess supports importing from a wide range of databases that include:
```

**Review by mattlord — APPROVED** [review]
LGTM! I only had minor notes that you can resolve as you feel is best. Thanks!

File: content/en/docs/17.0/overview/supported-databases.md (hunk 6 above)
Comment (GuptaManan100) [reply content/en/docs/17.0/overview/supported-databases.md]: Done

**Pull request merged**